test/Constraints/fixes.swift

```diff
@@ -71,16 +71,6 @@ func forgotOptionalBang(_ a: A, obj: AnyObject) {
   // expected-note@-2{{force-unwrap using '!' to abort execution if the optional value contains 'nil'}}
 }
 
-// Crash with one-element tuple with labeled element
-class Dinner {}
-
-func microwave() -> Dinner {
-  let d: Dinner? = nil
-  return (n: d) // expected-error{{value of optional type 'Dinner?' must be unwrapped to a value of type 'Dinner'}}
```

Comment:
What happened to this diagnostic?

Reply:
It got moved to test/Constraints/tuples.swift. It's not testing constraint system 'fixes' anymore, it's just a general ill-formed example using tuples.